types/hls.js/index.d.ts

```diff
@@ -1366,7 +1432,7 @@ declare namespace Hls {
     /**
      * levelDetails object
      */
-    details?: LevelDetails;
+    details?: string;
```

Comment:
Why did this type change?  If it was intentional, do the docs (e.g. immediately above) need to be updated?